benchmarks/conftest.py

```diff
@@ -0,0 +1,177 @@
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            # Quoto.date cannot be null.
+            row[3].value or datetime.now(),
+            # Keep micro values.
+            '{:f}'.format(PyDecimal(row[4].value)) if row[4].value is not None else None,
+            '{:f}'.format(PyDecimal(row[5].value)) if row[5].value is not None else None,
+            '{:f}'.format(PyDecimal(row[6].value)) if row[6].value is not None else None,
+            '{:f}'.format(PyDecimal(row[7].value)) if row[7].value is not None else None,
+            '{:f}'.format(PyDecimal(row[8].value)) if row[8].value is not None else None,
+        ))
+
+    h.reverse()
+    return h[:days]
+
+@pytest.fixture(scope='session')
+def bitcoin_quotes(days: int = 1246):
+    rows = list(wb['Bitcoin'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+            row[7].value,
+            row[8].value,
+        ))
+
+    h.reverse()
+    return h[:days]
+
+@pytest.fixture(scope='session')
+def intraday_quotes(days: int = 1564):
+    rows = list(wb['Intraday'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+            row[7].value,
+            row[8].value,
+        ))
+
+    h.reverse()
+    return h[:days]
+
+@pytest.fixture(scope='session')
+def longish_quotes(days: int = 5285):
+    rows = list(wb['Longish'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+            row[7].value,
+            row[8].value,
+        ))
+
+    h.reverse()
+    return h[:days]
+
+@pytest.fixture(scope='session')
+def longest_quotes():
+    rows = list(wb['Longest'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+            row[7].value,
+            row[8].value,
+        ))
+
+    return h
+
+@pytest.fixture(scope='session')
+def penny_quotes():
+    rows = list(wb['Penny'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[2].value,
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+            row[7].value,
+        ))
+
+    return h
+
+@pytest.fixture(scope='session')
+def zigzag_quotes(days: int = 342):
+    rows = list(wb['ZigZag'])[1:]
+
+    h = []
+    for row in rows:
+        h.append(Quote(
+            row[1].value,
+            row[2].value,
+            row[3].value,
+            row[4].value,
+            row[5].value,
+            row[6].value,
+        ))
+
+    h.reverse()
+    return h[:days]
+
+@pytest.fixture(scope='session')
+def converge_quantities():
```

Comment:
You probably don't need these static `convergence_quantities` numbers here.  These are for the convergence tests in the source .NET library.  They are not functional tests; but rather, tests to see when indicator values converge to accuracy.  Since this only needs to be done in the source library; you won't need to reproduce those Convergence tests here.

Reply:
Thanks for letting me know